src/borrower/collect/collector.sol

```diff
@@ -72,26 +72,33 @@ contract Collector is DSNote, Auth {
         wards[msg.sender] = 1;
     }
 
-    function depend(bytes32 what, address addr) external auth {
-        if (what == "distributor") distributor = DistributorLike(addr);
-        else if (what == "shelf") shelf = ShelfLike(addr);
-        else if (what == "pile") pile = PileLike(addr);
-        else if (what == "threshold") threshold = RegistryLike(addr);
+    /// sets the dependency to another contract
+    function depend(bytes32 contractName, address addr) external auth {
+        if (contractName == "distributor") distributor = DistributorLike(addr);
+        else if (contractName == "shelf") shelf = ShelfLike(addr);
+        else if (contractName == "pile") pile = PileLike(addr);
+        else if (contractName == "threshold") threshold = RegistryLike(addr);
         else revert();
     }
 
-    // --- Collector ---
+    /// sets the selling-price of an NFT
     function file(uint loan, address buyer, uint nftPrice) external auth {
         require(nftPrice > 0, "no-nft-price-defined");
         options[loan] = Option(buyer, nftPrice);
     }
 
+
+    /// if a loan debt is above the loan threshold the nft should be sold
+    /// therefore the ownership of the nft is transferred to the collector
     function seize(uint loan) external {
         uint debt = pile.debt(loan);
         require((threshold.get(loan) <= debt), "threshold-not-reached");
         shelf.claim(loan, address(this));
     }
 
+
+    /// a nft can be collected if the collector is the nft- owner
+    /// which implies the debt was above the loan threshold at certain point in time
```

Comment:
which implies the debt was above the loan threshold at certain point in time and then seized by the collector

Reply:
👍

Reply:
I think I would use 'An NFT' for this. The rules are not super clear cut on this but it rolls of the tongue more easily.

 https://www.quora.com/Why-is-an-used-instead-of-a-before-the-word-NGO-even-though-N-is-not-a-vowel?share=1

Reply:
Capitalize NFT and use "An NFT"

Reply:
I would remove the "implies" part. You can say, to:
```suggestion
    /// The NFT needs to be `seized` first to transfer ownership to the collector.
```